tox.ini

```diff
@@ -2,8 +2,9 @@
 envlist =
     py{37,38,39,310}-dj32
     py{38,39,310}-dj40
-    py{38,39,310,py311}-dj41
-    py{38,39,310,py311}-dj42
+    py{38,39,310,311}-dj41
+    py{38,39,310,311}-dj42
+    py{310,311,312}-dj42
```

Comment:
Should this be `py{310,311,312}-dj50`?

Reply:
🤦 Yes, oops, fixed here: https://github.com/jazzband/django-embed-video/pull/204/commits/96bae53fa73970e0228e72561ffd072a91a31963